src/uiprotect/data/devices.py

```
@@ -3589,10 +3666,92 @@ def callback() -> None:
+            BadRequest: If the camera is not paired with this chime
+
+        """
+        # Find the current ring setting for this camera
+        ring_setting = None
+        for setting in self.ring_settings:
+            if setting.camera_id == camera.id:
+                ring_setting = setting
+                break
+
+        if ring_setting is None:
+            raise BadRequest(f"Camera {camera.id} is not paired with chime")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)